<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(71, 105, 127);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QTextEdit" name="textEdit">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="pushbutton">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(206, 206, 206);</string>
     </property>
     <property name="text">
      <string>Translate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>persian to english</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="textEdit_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>english to persian</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="clearbutton">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(206, 206, 206);</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
